python/ray/tests/test_enforce_autoscaler_proto_stability.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+
+import pytest
+
+
+def test_enforce_autoscaler_proto_stability():
```

Comment:
i don't really see the purpose of this test. isn't it going to be triggered exactly the same times the codeowners gets applied?

Reply:
Just an extra complaint for a bit of extra safety on the parts that threaten autoscaler compatibility.

Reply:
We've seen that the comment in the file is not enough -- there should be an actual CI failure.